core/src/main/java/edu/wpi/grip/core/sources/IPCameraFrameGrabber.java

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
Can you add very clear comments on the lines you've added/changed to be completely clear what is different.
Do it inline like `// Our change`